src/segments/nx.go

```diff
@@ -0,0 +1,55 @@
+package segments
+
+import (
+	"encoding/json"
+	"fmt"
+	"oh-my-posh/environment"
+	"oh-my-posh/properties"
+)
+
+type Nx struct {
+	language
+}
+
+func (a *Nx) Template() string {
+	return languageTemplate
+}
+
+func (a *Nx) Init(props properties.Properties, env environment.Environment) {
+	a.language = language{
+		env:        env,
+		props:      props,
+		extensions: []string{"workspace.json"},
+		commands: []*cmd{
+			{
+				regex: `(?:(?P<version>((?P<major>[0-9]+).(?P<minor>[0-9]+).(?P<patch>[0-9]+))))`,
+				getVersion: func() (string, error) {
```

Comment:
Can you define this function below and reference it here?

Reply:
Done.  I'm not really proficient with Go, but it seems to work.

I basically just copy/pasted from the Angular segment. 😅

Reply:
I noticed that only after the facts. And truth be told, that was was actually not OK. I must've been a bit under the weather when I merged that one in. But, fixed it.